package.json

```diff
@@ -85,6 +75,13 @@
       "type": "object",
       "title": "Nushell IDE Support",
       "properties": {
+        "nushellLanguageServer.implementation": {
+          "scope": "window",
+          "type": "string",
+          "enum": ["extension", "nu --lsp", "nuls"],
+          "default": "extension",
+          "description": "Switch between different language server implementations (requires restarting Visual Studio Code to take effect)."
+        },
```

Comment:
Bah, this is the only deliberate change here

The rest is auto-`prettier`, which I can revert if necessary

Reply:
If you're using vscode to edit these files, one thing that I've just learned is you can do ctrl/cmd-shift-p and choose `File:Save without formatting` and it stops things like prettier or other formatters from running on save.